### PR TITLE
Mk list item style

### LINKED ISSUE #63: Style <ListItem> component
Follow figma drawings from @N-Zubko! https://www.figma.com/file/oPXbZdYLzuHU0bya9SHdET/Pac-man-styled-app?node-id=9%3A129

### PR DESCRIPTION
**Pull request by mira-kine**
## Description

This PR styles the `ListItem` aka the items that come up in each list according to its time frame. I used flex box to align the items in columns under the titles of "Item", "Bought" and "Delete", and put the corresponding ghost colors for the time frames that were used in `AddItem` in the title of each section. 

## Related Issue

Closes #63 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
|   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |
|  ✓   | 💅 : Styling              |

### Before
Screen Shot 2022-08-27 at 9 00 43 PM

### After

Screen Shot 2022-08-27 at 8 55 08 PM

## Testing Steps / QA Criteria
1. git checkout main then git pull
2. git checkout MN-list-item-style
3. npm run start
4. Navigate to your List page and checkout list items
5. add different list items with different time frames to check for organization and responsiveness

### CODE REVIEW

**Comment by N-Zubko** [comment]
I have only UI/UX suggestions.
1. Currently even empty time frames contain text information and a ghost icon. This is confusing - they contain no items but the labels and the icons create an illusion of not empty.
* If it is possible, could we not render empty boxes? I cannot give a solution from the top of my head, I will look at the code and think.
* Or could we remove the text ( Item - Bought - Delete ), because the app is intuitive and the labels only distract?

2. Also for Bought/Purchased - could we add back a tick to this checkbox?

Screen Shot 2022-08-27 at 10 27 33 PM

**Comment by michellerenehey** [comment]
> 

Oh man! I spent about an hour on that conditional rendering idea tonight and it is TRICKY because we are nesting .maps to get the data! (We get the groups.filteredData deep inside of the first map... so to tell whether or not there is data at all I think we'd need to go inside two layers of a map _before_ rendering). I'll bet it's possible, but I'm not sure it is going to be possible for us to do it before our presentation tomorrow. 😞 I feel ok with the labels being there even if there aren't items, but understand if others want to change it!

I've added the checkmark tick into 'bought'!

**Review by michellerenehey — APPROVED** [review]
Approving this! Excited to merge into the <List/> component and also VERY SCARED.

**Review by N-Zubko — APPROVED** [review]

**Comment by N-Zubko** [comment]
> > 
> 
> Oh man! I spent about an hour on that conditional rendering idea tonight, @N-Zubko, and it is TRICKY because we are nesting .maps to get the data! 

I know! Well, if during the demonstration tomorrow we add items to all categories and have something in the list already old enough fall under the inactive category, nobody's gonna notice the empty boxes 😈

File: src/views/List.jsx [hunk 1]
```diff
@@ -134,26 +150,36 @@ export function List({ data, listToken, loading, confirmLogOut }) {
...
-											listToken={listToken}
-										/>
-									);
-								})}
-						</section>
-					);
-				})}
+				<div className="group-container">
+					{/* filter through groups array for each group to display by time frame */}
+					{groups.map((group) => {
```
Comment (Trevor-Rezac) [inline src/views/List.jsx:155]: I think if we add ```data.length === 0 ? <></> :``` here on line 155 right before groups.map, I think this should remove the group renderings when there are no items. Then, when an item is added, all groups should appear and the item will be in the appropriate group.

**Review by Trevor-Rezac — CHANGES_REQUESTED** [review]

File: src/views/List.jsx (hunk 1 above)
Comment (Trevor-Rezac) [reply src/views/List.jsx:155]: Without items added:

Screen Shot 2022-08-28 at 10 15 15 AM

With an item added:
Screen Shot 2022-08-28 at 10 16 03 AM

**Comment by Trevor-Rezac** [comment]
Great job with this styling Mira! The centering of checkbox and delete button looked tricky! Glad Michelle was able to help figure that out! 

I agree with Nadia about not rendering the groups when there are no list items so I recommended one change! If for some reason it doesn't end up working, please feel free to disregard.

File: src/views/List.jsx [hunk 2]
```diff
@@ -40,27 +45,38 @@ export function List({ data, listToken, loading, confirmLogOut }) {
...
+
 			filteredData: (item) => {
 				return item.currentEstimate >= 30 && item.currentEstimate < 60;
 			},
 		},
 		{
 			timeFrame: 'Inactive',
 			subLabel: '60 days or more',
+			image: (
+				<img className="blinkies" src={blueblinky} alt="blue-blinky logo" />
```
Comment (lindseyindev) [inline src/views/List.jsx:78]: These look so gooooood on the list!!!

File: src/views/List.jsx [hunk 3]
```diff
@@ -134,26 +150,36 @@ export function List({ data, listToken, loading, confirmLogOut }) {
...
+							<section className={group.timeFrame}>
+								<div className="group-title">
+									<span>{group.image}</span>
+									<h1>{group.timeFrame}</h1>
+								</div>
+								<div className="list-titles">
+									<h3 className="item-title-1">Item</h3>
+									<h3 className="item-title-2">Bought</h3>
+									<h3 className="item-title-3">Delete</h3>
+								</div>
```
Comment (lindseyindev) [inline src/views/List.jsx:166]: Nice work handling the layout with this section, it looks amazing and great work on updating the groups config to include additional display information.

I saw some of the solutions you did to get this to all line up and I would recommend an HTML table for this layout and I think that would've helped with some of the alignment but honestly great work on figuring out how to get the alignment you wanted through CSS which is a huge skill

**Review by lindseyindev — APPROVED** [review]
Great work on this PR! I really like the way you updated the groups and it's wild to me how much the look and feel of the app is changing and just getting even more in theme through each of these PRs!

**Review by Trevor-Rezac — APPROVED** [review]
I misunderstood the issue and thought it was a quick fix to render the groups. Approving this PR.

File: src/views/List.jsx (hunk 1 above)
Comment (nickytonline) [reply src/views/List.jsx:155]: I like that suggestion @Trevor-Rezac. 👏🏻

**Pull request merged**